packages/squiggle-lang/src/rescript/Reducer/Reducer_Peggy/Reducer_Peggy_GeneratedParser.peggy

```diff
@@ -0,0 +1,346 @@
+// Try in https://peggyjs.org/online
```

Comment:
I might be misunderstanding this, but, my impression is that this file is handwritten, not generated. Is that the case? If so, then why is this called "GeneratedParser"? Is that because this file is used to generate the parser?